develop/EPM_Integration_Training/Other_Options/Other_Options.md

```diff
@@ -0,0 +1,37 @@
+---
+uid: EpmIntegrationTrainingOtherOptions
+---
+
+# Other configuration options
+
+## Titlebox Data section
+
+The native look of the Data section of an EPM Entity is two columns filled with all of the information in two columns related to the row of the entity in the column order of the table. All columns that are shown are also shown in the Data section. If a column has width=0, then it will not show in this section and will be unable to be shown.
+
+To further configure the look of the data section and also add title boxes, you need to create a page called CPEIntegration/[EPM System Type] i.e. CPEIntegration/Region.
+
+On this page, you may add parameters as you would any other page, but you may also add individual column parameters associated with the entity. If using duplicateAs option, both the native and view table column will occupy the same space but will not cause issues in DataMiner.
+
+![EPM Data layout](~/develop/images/EPM_Data_Layout.png)
+
+![EPM Data page](~/develop/images/EPM_Data_page.png)
+
+## EPM configuration
+
+There are a couple of options that are required for the EPM protocol to have so the DMS knows which element is the front end of the system. First is a parameter named “Elementmanagertype” and the other is having the Protocol.Display@type attribute set to "element manager". More information can be found here: [EPM Integration in Cube](xref:AdvancedEpm)
```

Comment:
The RN for this feature ([21798](https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm.aspx?ID=21798)) is still indicated as on hold and in soft launch. Do you know if the CPEIntegration soft-launch option is required for this as well?

Reply:
This feature/configuration does not require the soft launch option, but won't really be utilized unless paired with the softlaunch option

Reply:
Do you mean that this is only useful if you use the Topology app?

Reply:
I believe so, I can't think of any reason to have those tags if you are not using the topology app. Maybe you can use the EPM Feed component in WebApps, but can't think of any other benefit